src/interface/src/app/map/map-manager.ts

```diff
@@ -679,9 +679,8 @@ export class MapManager {
 
     map.dataLayerRef?.remove();
 
-    let filepath = map.config.dataLayerConfig.filepath;
     let layer = map.config.dataLayerConfig.layer;
-    if (filepath?.length === 0 || !filepath) return;
```

Comment:
I think by doing this, we're starting to signal the end of filepath, correct?  I.e. filepath is no longer required.

Reply:
Yeah I don't think it's needed anywhere in the map code. As far as I can tell the only other place where layer filepaths are still used are in the computer_conditions file (which I'm pretty sure is obsolete with the new data) and as the keys in the condition metadata dictionary